docs/tabular_buckets_create_table.msc

```diff
@@ -0,0 +1,128 @@
+msc {
+    width=1800;
+    C [label="client"],
+    KVA [label="riak_kv_ts_api"],
+    CBT [label="riak_core_bucket_type"],
+    CC [label="riak_core_claimant"],
+    CM [label="riak_core_metadata"],
+    CMM [label="riak_core_metadata_manager"],
+    KVML [label="riak_kv_metadata_store_listener"],
+    KVNT [label="riak_kv_ts_newtype"];
+
+    C -> KVA [label="create table"];
+    KVA -> CBT [label="create"];
+    CBT -> CC [label="create_bucket_type"];
+    CC -> CC [label="claimant"];
+    CC -> CC [label="create_bucket_type"];
+    CC -> CC [label="get_bucket_type"];
+    CC -> CM [label="get"];
+    --- [label="bucket DNE"];
+    CC <- CM [label="<- get, undefined"];
```

Comment:
Yes, <- means return, either as a literal function return or a return message (cast).

CC call get_bucket_type on itself which contains a call from CC to get  on CM. This is the flow of things. Including the self call to get_bucket_type could be elided, but IMO helps in the understanding of what CM's get is for.